src/tutorials/media_library.md

```diff
@@ -0,0 +1,221 @@
+---
+id: media_library
+name: Uploading Images
+title: Uploading Images
+tags:
+    - Tutorial
+---
+<!-- vale Manticore.FirstPerson = NO -->
+# Uploading Images using the Media Library
+
+![!Preview](../img/MediaLibraryTutorial/preview.png){: .center loading="lazy" }
+
+## Overview
+
+In this tutorial, you will learn how to use the [Media Library](../references/media_library.md) in Core to upload images to your account from within the Core editor that can be used in any of your projects.
+
+Being able to upload images allows you to create UI (User Interface) that is unique for your game that can be used in screen space and world space which can help make your game stand out and improve the player experience.
+
+The focus of this tutorial will be the Media Library and creating brushes from images you have uploaded to the Media Library. Interactive functionality will not be covered, for example, making buttons the player can click on.
+
+* **Completion Time:** ~20 minutes
+* **Knowledge Level:** No previous knowledge.
+* **Skills you will learn:**
+    * Where to find the Media Library.
+    * How to upload images to the Media Library.
+    * Creating image brushes from content in your Media Library.
+    * Using image brushes in the UI to create a shop.
+
+## Download Image Pack
+
+For this tutorial, you can use your images if you prefer, or if you want to follow along to recreate the shop UI, then you will need to download an image pack such as the one linked below. The pack below will require you to cut or slice the parts you want to use.
+
+[Mobile Game GUI](https://graphicburger.com/mobile-game-gui/)
+
+Always check the license that comes with any images you download that you will be using in your project. For the above pack, the license is included with the download.
+
+## Uploading Images to the Media Library
+
+To use images in any of your projects, the images need to be uploaded to your Media Library account first. Once the images have been uploaded, you can then create brushes that can be used in the UI.
+
+### Open Media Library
+
+To open the Media Library, go to the **Window** menu at the top of your Core Editor and select **Media Library** from the list.
+
+![!Window Menu](../img/MediaLibraryTutorial/window_menu.png){: .center loading="lazy" }
+
+### Uploading Images
+
+From the Media Library window, select the **Upload Image** button and upload all the images you want to use for your UI.
+
+In the image below, all the images that will be used in this tutorial have been uploaded.
+
+![!Media Library](../img/MediaLibraryTutorial/media_library.png){: .center loading="lazy" }
+
+## Creating Brushes
+
+Before you can use an image in your project, a custom brush needs to be created. A brush is a usable asset that references the image on your account.
```

Comment:
"needs to be" is passive voice. I would go ahead and and say "you will need to create". Also cover here that we're calling it both brush (is that really lower case?) and Image Asset.

Reply:
It's "Brush" (capitalized) in properties panel of a brush asset.